CODING_STANDARD

```diff
@@ -171,6 +171,20 @@ Output:
   (status(), error(), debug(), etc)
 - Use '\n' instead of std::endl
 
+Unit tests:
+ - Unit tests are written using Catch: https://github.com/philsquared/Catch/
+ - For large classes:
+    - Each file should contain tests for one function in one class
```

Comment:
Suggested clarity rewording:
Create a separate file for each method of each class that contains the tests for that method